.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: Bug Report 🐛
 about: Something isn't working as expected? Here is the right place to report.
+labels: bug, to be triaged
```

Comment:
`to be triaged` doesn't need to be surrounded by quotes does it? I couldn't find documentation on that specifically.

Reply:
I couldn't either, I just know that it needs to be comma separated. I'm going to merge this and see if it works.